Add etcd-client-urls flag in backend reference (#2006)
sensu/sensu-docs: content/sensu-go/5.16/reference/backend.md

      --etcd-advertise-client-urls strings         list of this member's client URLs to advertise to the rest of the cluster. (default [http://localhost:2379])
      --etcd-cert-file string                      path to the client server TLS cert file
      --etcd-cipher-suites strings                 list of ciphers to use for etcd TLS configuration

>       --etcd-client-urls                           list of urls to listen on for client traffic when sensu-backend is not operating as an etcd member

Reviewer: ```suggestion
      --etcd-client-urls string                    client URLs to use when operating as an etcd client
```

etcd-client-cert-auth: true{{< /highlight >}}

| etcd-client-urls      |      |
------------------------|------

> description             | List of URLs to listen on for client traffic when a sensu-backend is not operating as an etcd member. Use with sensu-backend and [sensu-backend init][20]. Returns a client that connects to an external etcd. If you do not use this flag, sensu-backend and sensu-backend-init will fall back to [--etcd-listen-client-urls][21].

Reviewer: ```suggestion
description             | List of client URLs to use when a sensu-backend is not operating as an etcd member. To configure sensu-backend for use with an external etcd instance, use this flag in conjunction with `--no-embed-etcd` when executing sensu-backend start or [sensu-backend init][20] . If you do not use this flag when using `--no-embed-etcd`, sensu-backend start and sensu-backend-init will fall back to [--etcd-listen-client-urls][21].
```

> description             | List of URLs to listen on for client traffic to use when a sensu-backend is not operating as an etcd member. Use with sensu-backend and [sensu-backend init][20]. Returns a client that connects to an external etcd. If you do not use this flag, sensu-backend and sensu-backend-init will fall back to [--etcd-listen-client-urls][21].

Author: ```suggestion
description             | List of URLs to listen on for client traffic when a sensu-backend is not operating as an etcd member. Use with sensu-backend and [sensu-backend init][20]. Returns a client that connects to an external etcd. If you do not use this flag, sensu-backend and sensu-backend-init will fall back to [--etcd-listen-client-urls][21].
```
